sass/login/keycloak-overrides.scss

```diff
@@ -96,3 +99,17 @@
   color: $color-primary;
   text-decoration: none;
 }
+
+// Stop language menu from creating unnecessary padding before title
+// and logo.
+// Set header as relative, so language menu can be positioned absolutely
+// in terms of it.
+.login-pf-header {
+  position: relative;
+}
+// Position language menu absolutely in the top right corner of header.
+#kc-locale {
+  position: absolute;
+  top: 0;
+  right: 0;
+}
```

Comment:
Do you have an autolinter installed?

Reply:
Yes, but I don't have one configured for this project, which is why that pesky newline is missing.